.github/workflows/main.yml

```
@@ -19,13 +19,26 @@ jobs:
           pip install .[dev]
           pip install coverage
 
+      - name: Build Docker image
+        run: docker build -t testnew:latest .
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
+        env:
+          TRIVY_DISABLE_VEX_NOTICE: "true"
+        with:
+            image-ref: testnew:latest
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)